examples/cryptocurrency-advanced/backend/src/transactions.rs

```diff
@@ -65,44 +66,62 @@ impl From<Error> for ExecutionError {
     }
 }
 
-transactions! {
-    /// Transaction group.
-    pub WalletTransactions {
-
-        /// Transfer `amount` of the currency from one wallet to another.
-        struct Transfer {
-            /// `PublicKey` of receiver's wallet.
-            to:      &PublicKey,
-            /// Amount of currency to transfer.
-            amount:  u64,
-            /// Auxiliary number to guarantee [non-idempotence][idempotence] of transactions.
-            ///
-            /// [idempotence]: https://en.wikipedia.org/wiki/Idempotence
-            seed:    u64,
-        }
+/// Transfer `amount` of the currency from one wallet to another.
+#[derive(Serialize, Deserialize, Clone, Debug, ProtobufConvert)]
+#[protobuf_convert("proto::Transfer")]
+pub struct Transfer {
+    /// `PublicKey` of receiver's wallet.
```

Comment:
Why aren't these comments present in the `proto` file?

Reply:
Can the rustdoc handle comments in the derive macroses?